trb_template.tex

```diff
@@ -1,365 +1,212 @@
-
-For R-Studio users using Sweave and .rnw files, you may enable shell escape command in the Global Options > Sweave settings.  Moreover, if your computer does not have a Perl interpreter you will need one, such as the ActivePerl, for the wordcount to work properly.
-
-\newpage
-
-\bibliographystyle{trb}
-\bibliography{trb_template}
-
-% End line numbering
-\nolinenumbers
-\end{document}
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbunofficial}
+\usepackage{graphicx}
+\usepackage[colorlinks=true,linkcolor=blue,citecolor=blue]{hyperref}
+
+% Put here what will go to headers as author
+\AuthorHeaders{Pritchard, Macfarlane, and Wang}
+\title{A \LaTeX\ Template for Papers Submitted to the Transportation
+Research Board}
+
+% TODO: add macros for easier formatting of \author.
+\author{%
+  \textbf{David Pritchard}\\
+  davidpritchard.org\\
+  \hfill\break% this is a way to add line numbering on empty line
+  \textbf{Gregory S. Macfarlane, Ph.D.}\\
+  \hfill\break%
+  \textbf{Chieh (Ross) Wang, Ph.D.}\\
+  cw@crosswang.org
+}
+
+% If necessary modify the number of words per table or figure default is set to
+% 250 words per table and figure
+% \WordsPerTable{250}
+% \WordsPerFigure{250}
+
+% If words are counted manually, put that number here. This does not include
+% figures and tables. This can also be used to avoid problems with texcount
+% program i.e. if one does not have it installed.
+% \TotalWords{200}
+
+\begin{document}
+\maketitle
+
+\section{Abstract}
+
+The Transportation Research Board (TRB) has unique and seemingly arbitrary
+requirements for manuscripts submitted for review. These requirements make it
+difficult to write the manuscripts quickly, and no existing \LaTeX\ style comes
+close to fooling the guidelines. This represents an initial effort at creating
+a template to meet the requirements of TRB authors using \LaTeX, R, Sweave,
+and/or other literate programming software.
+
+\hfill\break%
+\noindent\textit{Keywords}: Keyword1, Keyword2
+\newpage
+
+\section{Introduction}
+The \trbcite{TRBGuide} has unique and somewhat arbitrary requirements for
+papers submitted for review and publication. While the initial submission is
+required to be in PDF format, submissions for publication in Transportation
+Research Record must be in Microsoft Office format. On top of this, the
+manuscripts must be line-numbered, captions are bolded and employ atypical
+punctuation, and the references must be numbered when cited and then printed in
+order. More details about the manuscript details can be found
+online~\footnote{\url{http://onlinepubs.trb.org/onlinepubs/AM/InfoForAuthors.pdf}}.
```

Comment:
TRB does not allow footnotes.  I'd change it to: +online at~\url{http://onlinepubs.trb.org/onlinepubs/AM/InfoForAuthors.pdf}.